<launch>
  <node name="edge_detector" pkg="ros_edge_detector" type="edge_detector.py" output="screen" >
    <param name="topic_sub" value="/controled_stream" />
    <param name="topic_pub" value="/detected" />
  </node>
</launch>
